pkgs/tools/networking/dd-agent/default.nix

```diff
@@ -26,61 +33,158 @@ let
     # due to flake8
     doCheck = false;
   };
+  version = "5.13.2";
+
+  integrations = fetchFromGitHub {
+    owner = "datadog";
+    repo = "integrations-core";
+    rev = version;
+    sha256 = "1nbjmkq0wdfndmx0qap69h2rkwkkb0632j87h9d3j99bykyav3y3";
+  };
+
+  jmxArtifact = fetchurl
+    { url = "http://dd-jmxfetch.s3.amazonaws.com/jmxfetch-0.13.0-jar-with-dependencies.jar";
+      sha256 = "02rfn17q42kxiv2sv306ibdhvi8l4sbjdwhh1nwdlxw46pqh4ks1";
+    };
 
 in stdenv.mkDerivation rec {
-  version = "5.11.2";
   name = "dd-agent-${version}";
 
   src = fetchFromGitHub {
     owner  = "datadog";
     repo   = "dd-agent";
     rev    = version;
-    sha256 = "1iqxvgpsqibqw3vk79158l2pnb6y4pjhjp2d6724lm5rpz4825lx";
+    sha256 = "0x2bxi70l2yf0wi232qksvcscjdpjg8l7dmgg1286vqryyfazfjb";
   };
 
   buildInputs = [
     python
     unzip
     makeWrapper
-    pythonPackages.requests
-    pythonPackages.psycopg2
-    pythonPackages.psutil
```

Comment:
The idea behind the removal of these was that they are not strictly necessary. They were simply the "defaults". NixOS service happens to provide these defaults but there's nothing stopping the user from adding these and any other packages back into dd-agent with an override whereas it's much more difficult to opt out of these from the service side of things.

Having said that maybe dd-agent expression itself should be more configurable with these things enabled by default and the service could flip things that way. Seems ugly to me though…